lms/djangoapps/courseware/user_state_client.py

```diff
@@ -41,6 +39,9 @@ class DjangoXBlockUserStateClient(XBlockUserStateClient):
         even though the actual stored state in the database will be ``"{}"``).
     """
 
+    # Use this sample rate for DataDog events.
+    API_DATADOG_SAMPLE_RATE = 0.01
```

Comment:
Yeah - I totally guesstimated the sample rate based on the XModule rendering DD events:

https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/x_module.py#L44

I figured that problem answers were probably happening around 10x more than those renders. A configuration item would require a release/hotfix to update (95% sure of that). But there's also a DB settings system - I'll look into that.